pkg/controller/controller.go

```diff
@@ -95,6 +96,12 @@ type topicKey struct {
 	name string
 }
 
+// A Δfn represents a pair of functions involved in an update
+type Δfn struct {
```

Comment:
How on earth do I type a Δ? Although I find it pretty cool, I think this doesn't work in practice. We need to think about users with a non US keyboard layout

Reply:
On Mac, it's alt+J. It's not US keyboard specific - I have a UK keyboard. ;-)

I expected this reaction BTW, but I was encouraged by Onsi's use of Ω (alt+Z) in Gomega. :-)

If you still feel uncomfortable, I'l refactor `∆fn` to `deltaFn`.

Reply:
I looked it up, and I tried it. It is US/UK specific, believe me, it doesn't work on a french layout (it does when I switch to US). I really think we should switch, sadly

Reply:
@ericbottard Actually, I've repented. I'll push an update soon.